AtelierTomato.Markov/AtelierTomato.Markov.Data/InMemorySentenceAccess.cs

```diff
@@ -0,0 +1,74 @@
+﻿using AtelierTomato.Markov.Data.Model;
+
+namespace AtelierTomato.Markov.Data
+{
+	public class InMemorySentenceAccess : ISentenceAccess
+	{
+		private readonly Random random = new Random();
+		public List<Sentence> SentenceStorage = [];

```

Comment:
oh yeah this is still public. you should never* have public fields, so make this private. honestly exposing this outside is still very questionable, check earlier review about my suggestion with internal - but we can just make a public property that exposes this with only as much as absolutely needed for the tests for now